src/containers/Party.tsx

```diff
@@ -49,9 +49,15 @@ class Party extends React.Component<any, any> {
 
 function mapStateToProps(state) {
     return {
-        party: state.party
+        party: state.party,
+        selfId: state.user.id,
+        hostId: state.party &&
+        state.party.users &&
+        state.party.users.find(u => u.host) &&
+        state.party.users.find(u => u.host).id
     };
 }
 
+const mapDispatchToProps = dispatch => bindActionCreators(ActionCreators as any, dispatch);
 
-export default connect(mapStateToProps)(Party);
\ No newline at end of file
+export default connect(mapStateToProps, mapDispatchToProps)(Party);
```

Comment:
I'm not sure, but I guess this logic will be repeated through multiple components, we can leave it as is now, and address any refactoring later